scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,243 @@
+    text-shadow: none;
+    text-transform: uppercase;
+    vertical-align: middle;
+
+    &:hover {
+      background: darken($brand-primary, $crm-darken-percentage);
+    }
+
+    &:active,
+    &:focus {
+      outline: none;
+    }
+
+    &.cancel {
+      background: $crm-white;
+      border: 1px solid $crm-copy;
+      color: $crm-cancel-color;
+
+      &:hover {
+        background: $crm-copy;
+        color: $crm-white;
+      }
+    }
+  }
+
+  .crm-actions-ribbon {
+    margin-bottom: 20px;
+  }
+
+  .crm-form-block.crm-form-block>.crm-submit-buttons,
+  .crm-form-block .form-item>.crm-submit-buttons {
+    border-top: 0;
+    margin-top: 0 !important;
+  }
+
+  #newLocation {
+    padding: 0 20px 20px 15px;
+
+    #crm-submit-buttons {
+      padding-left: 5px;
+    }
+
+    .crm-event-is_show_location br {
+      display: none;
+    }
+
+    h3 {
+      @include section-title;
+      margin: 0 -20px 20px -15px;
+    }
+  }
+
+  .crm-event-manage-location-form-block > table {
+    margin-left: 15px;
+
+    .status {
+      padding-left: 0;
+    }
+  }
+
+  #priceSet .message {
+    margin-right: 15px;
+  }
+
+  #contributionType .description {
+    padding-right: 15px;
+  }
+
+  fieldset#discount,
+  fieldset#map-field {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    > legend {
+      @include section-title;
+      display: block;
+      width: 100%;
+      margin-left: -20px;
+    }
+
+    table {
+      min-width: 100%;
+    }
+
+    tr {
+      background-color: transparent;
+      border-top: 1px solid $crm-table-border-color;
+      display: block;
+      margin: 0 -20px;
+      padding: 0px 20px 0px 20px;
+    }
+
+    td {
+      border: 0;
+      display: inline-block;
+      padding: 5px 10px 9px;
+      width: 260px;
+    }
+
+    .dateplugin {
+      margin-top: 5px;
+    }
+
+    .columnheader {
+      display: block;
+      background-color: $gray-lighter;
+      border-bottom: 1px solid $crm-grayblue-dark;
+      border-left: 0;
+      border-radius: 0;
+      border-right: 0;
+      margin: 20px -20px 0;
+      padding: 6px 20px 4px 20px;
+
+      br {
+        display: none;
+      }
+
+      td {
+        color: $crm-columnheader-title-color;
+        font-size: 13px;
+      }
+
+      label {
+        margin-left: 5px;
+      }
+    }
+  }
+
+  fieldset#discount {
+    .columnheader td:first-child,
+    .form-item td:first-child {
+      padding-left: 2px;
+      width: 20px;
+    }
+
+    .crm-form-submit {
+      margin: 10px 20px 0;
+    }
+  }
+
+  #isDiscount {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    .crm-event-manage-fee-form-block-is_discount td {
+      width: auto;
+    }
+  }
+
+  .crm-collapsible {
+    padding: 20px;
+
+    &.collapsed .collapsible-title {
+      box-shadow: $crm-accordion-shadow;
+
+      &:before {
+        content: $fa-chevron-right;
+      }
+    }
+
+    .collapsible-title {
+      background-color: $crm-collapsible-title-background;
```

Comment:
This is wrongly styled element please visit /civicrm/contact/add?reset=1&ct=Individual to see style of accordions.

Reply:
Styles cahaged
![accordion](https://user-images.githubusercontent.com/3272950/34302292-f38ebaf0-e738-11e7-9653-5c8ccee90033.png)